common/src/lib.rs

```diff
@@ -15,6 +15,7 @@ use tokio::sync::RwLock as TokioRwLock;
 
 pub mod default_wrapper;
 pub mod keyed_list;
+pub mod metrics_helpers;
```

Comment:
I'd suggest we make this its own crate. `common` is convenient, but it's likely that this functionality will outgrow the `common` crate, maybe `renegade-metrics`?